GunplaOnlineShop/GunplaOnlineShop/Views/Home/Index.cshtml

```diff
@@ -1,8 +1,8 @@
 ﻿@{
-    ViewData["Title"] = "Home Page";
+    ViewData["Title"] = "Hello Home";
```

Comment:
This is the Title of the page which will display as the name of the tab, 
![image](https://user-images.githubusercontent.com/24485149/84575534-f4989380-ad6a-11ea-8a20-53f778142dcf.png)
please use a proper name "xxx gunpla shop"